Automattic/jetpack
# Integrate VideoPress with Divi Builder

#### Changes proposed in this Pull Request:


* Adds very basic Divi Builder support.
* You can embed by guid or by video url.
* The video will grow to fit the container.
* No support for video privacy at this point. Videos need to be public for this to work.

Note: Originally made using [create-divi-extension](https://github.com/elegantthemes/create-divi-extension). Later the code was adapted to fit in the VideoPress/Jetpack build system and conventions.

#### Other information:


#### Jetpack product discussion



pxWta-1KB-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Run install and build for the plugin you are using, e.g. `jetpack install plugins/jetpack` and `jetpack build plugins/jetpack`. Note: You may need to build dependencies, at least for the first time. You can do that with e.g. `jetpack build --all`
* You will need to install Divi. If you are an Automattician and need Divi, refer to pxWta-1KB-p2 
* Create a new post and use the Divi builder.
* Create a layout and then search for "videopress". Select that.
* Test pasting both guids and urls.
* You can see a VideoPress embed
